pkg/controller/cheinstallation/cheinstallation_controller.go

```diff
@@ -194,6 +221,67 @@ func (r *ReconcileCheInstallation) ensureCheSubscription(logger logr.Logger, che
 	return false, nil
 }
 
+func (r *ReconcileCheInstallation) ensureWatchCheCluster() error {
```

Comment:
Let's add some comments about what we are doing with watchCheCluster. It may happen that CheCluster CRD is not installed yet (then we set watchCheCluster func), so we need to requeue till it's available, etc.

Reply:
Done https://github.com/codeready-toolchain/toolchain-operator/pull/17/commits/8cf2914a793fa90ffa448f261fa0e3cf3515064b